webapp/javascript/redux/reducers/filters.js

```diff
@@ -74,6 +74,22 @@ function decodeTimelineData(timelineData) {
   });
 }
 
+function decodeFlamebearer({ flamebearer, metadata, leftTicks, rightTicks }) {
+  const fb = {
+    ...flamebearer,
+    format: metadata.format,
+    spyName: metadata.spyName,
+    sampleRate: metadata.sampleRate,
+    units: metadata.units,
+  };
+  if (fb.format === 'double') {
+    fb.leftTicks = leftTicks;
+    fb.rightTicks = rightTicks;
+  }
+  fb.levels = deltaDiffWrapper(fb.format, fb.levels);
+  return fb;
```

Comment:
Can we somehow parse/indicate in the frontend that's a `version: 1` that we are dealing with?

Reply:
I added the version field to the flamebearer model in https://github.com/pyroscope-io/pyroscope/pull/596/commits/39fc8f04295f8374b3e37b21d6777d59c2a6ffef